### PR TITLE
ENV, CLI, Reporters

### PR DESCRIPTION
**Pull request by manatskaya**
Hi Liubov,
Please check when you have a time.
Thanks
Svitlana

### CODE REVIEW

File: .env
Comment (landriiuk) [inline .env:1]: Pls do not share your data from env file.

File: .env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+BASE_URL=https://demoqa.com/
+PASSWORD=QWERTY123456@
+USERNAME=Test12
+FIRSTNAME=Test User
+LASTNAME= Test User
```
Comment (landriiuk) [inline .env]:
```suggestion
BASE_URL=
PASSWORD=
USERNAME=
FIRSTNAME=
LASTNAME= 
```

File: env.example
Comment (landriiuk) [inline env.example:1]: this file should be called .env.example

**Review by landriiuk — APPROVED** [review]
Looks good. leaved comment about env file
Pls fix issues, merge this pr

File: .env
Comment (manatskaya) [reply .env:1]: Thank you for the review. Will update this PR with correct env. Probably I saved changes in .env files before I added it to .gitignore.

**Pull request merged**